docs/book/component-gallery/annotators/label-studio.md

```diff
@@ -48,6 +48,14 @@ and add it to your stack:
 zenml integration install label_studio
 ```
 
+{% hint style="warning" %}
+There is a known issue with Label Studio installations via `zenml integration
+install...`. You might find that the Label Studio installation breaks the ZenML
+CLI. In this case, please run `pip install 'pydantic==1.10.4'` to fix the issue
```

Comment:
Can we maybe include the full range of supported pydantic versions in this instruction as well? We don't actually need 1.10.4 for this to work I assume